tests/test_google_ads_interrupted_sync_full_table.py

```diff
@@ -0,0 +1,130 @@
+        # should have a bookmark value prior to the 'completed' streams.
+
+        interrupted_state = {
+            'currently_syncing': ('campaign_criterion', '5548074409'),
+            'bookmarks': {
+                'campaign_criterion': {'5548074409': {'last_pk_fetched': 16990616126}},
+           }
+        }
+
+        # set state for interrupted sync
+        menagerie.set_state(conn_id, interrupted_state)
+
+        # Run another sync
+        self.run_and_verify_sync(conn_id)
+
+        # acquire records from target output
+        interrupted_sync_records = runner.get_records_from_target_output()
+        final_state = menagerie.get_state(conn_id)
+
+        # stream-level assertions
+        for stream in streams_under_test:
+            with self.subTest(stream=stream):
+
+                # gather results
+                full_records = [message['data'] for message in full_sync_records[stream]['messages']]
+                full_record_count = len(full_records)
+                interrupted_records = [message['data'] for message in interrupted_sync_records[stream]['messages']]
+                interrupted_record_count = len(interrupted_records)
+
+                # campaign_criterion stream has a composite primary key.
+                # But, to filter out the records, we are using only campaign_id respectively.
+                if stream == "campaign_criterion":
+                    primary_key = "campaign_id"
+                else:
+                    primary_key = next(iter(self.expected_primary_keys()[stream]))
+
+                # Verify that all records in the full sync come in Ascending order.
+                # That means id of current record is greater than id of previous record.
+                for i in range(1, full_record_count):
+                    self.assertGreaterEqual(full_records[i][primary_key], full_records[i-1][primary_key], 
+                                            msg='id of the current record is less than the id of the previous record.')
+
+                # Verify that all records in the interrupted sync come in Ascending order.
+                # That means id of current record is greater than id of previous record.
+                for i in range(1, interrupted_record_count):
+                    self.assertGreaterEqual(interrupted_records[i][primary_key], interrupted_records[i-1][primary_key], 
+                                            msg='id of the current record is less than the id of the previous record.')
+
+                if stream in interrupted_state['bookmarks'].keys():
+                    
+                    # Verify second sync(interrupted_sync) have the less records as compared to first sync(full sync)
+                    self.assertLess(interrupted_record_count, full_record_count)
+                    
+                    # Verify id value of first record in interrupted sync is less than last_pk_fetched
```

Comment:
I think the comment and error msg in the assertion are contradicting the actual assertion being made.

Reply:
I have updated the comment according to the actual assertion.